File: Ex1Num8.cs
using System;

class Program
{
    static void Main()
    {
        int n;
        Console.Write("Введите n: ");
        n = int.Parse(Console.ReadLine());

        for (int i = 1; i <= n; i++)
        {
            for (char ch = (char)('A' + n - 1); ch >= 'A' + n - i; ch--)
            {
                Console.Write(ch + " ");
            }
            Console.WriteLine();
        }
    }
}

File: Ex1Num8.rs
use std::io;

fn main() {
    let mut input = String::new();
    println!("Введите n: ");
    
    io::stdin().read_line(&mut input).expect("Ошибка при чтении ввода");
    let n: usize = input.trim().parse().expect("Ожидалось целое число");

    for i in 1..=n {
        for ch in (b'A' + (n - i) as u8..=b'A' + (n - 1) as u8).rev() {
            print!("{} ", ch as char);
        }
        println!();
    }
}

